CRANIO subtype section added (#146)
AlexsLemonade/OpenPBTA-manuscript: content/03.methods.md

6. Non-MB and non-ATRT embryonal tumors with <i>CIC-NUTM1</i> or other <i>CIC</i> fusions, were subtyped as `CNS EFT-CIC` (CNS Ewing sarcoma family tumor with <i>CIC</i> alteration).
7. Non-MB and non-ATRT embryonal tumors that did not fit any of the above categories were subtyped as CNS Embryonal, NOS (CNS Embryonal tumor, not otherwise specified).

> Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (`CRANIO, ADAM`), papillary (`CRANIO, PAP`) or undetermined (`CRANIO, To be classified`) based on the following criteria:

Reviewer: ```suggestion
Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (`CRANIO, ADAM`), papillary (`CRANIO, PAP`) or undetermined (`CRANIO, To be classified`) based on the following criteria [@doi:10.3171/jns.1995.83.2.0206; @doi:10.3171/jns.1998.89.4.0547]:
```

> 1. Craniopharyngiomas with BRAF p.V600E mutation in patients age over 40-years-old were subtyped as `CRANIO, PAP`.

Reviewer: ```suggestion
1. Craniopharyngiomas from patients over 40 years old with a _BRAF_ p.V600E mutation were subtyped as `CRANIO, PAP`.
```

> 2. Craniopharyngiomas with mutations in exon 3 of _CTNNB1_ in patients age below 40-years-old were subtyped as `CRANIO, ADAM`.

Reviewer: ```suggestion
2. Craniopharyngiomas from patients younger than 40 years old with mutations in exon 3 of _CTNNB1_ were subtyped as `CRANIO, ADAM`.
```

> 3. Craniopharyngiomas that does not fall into the above two categories were subtyped as `CRANIO, To be classified`. 

Reviewer: ```suggestion
3. Craniopharyngiomas that do not fall into the above two categories were subtyped as `CRANIO, To be classified`. 
```

> The age of the patients were added as criteria for subtyping since papillary craniopharyngiomas (PCP) occur almost exclusively in adults and adamantinomatous craniopharyngiomas (ACP) occur mostly in childhood or young adolescence [@doi:10.3171/jns.1995.83.2.0206; @doi:10.3171/jns.1998.89.4.0547]. 

Reviewer: I would probably remove this – I think it's too much detail here – and then move the citations up (will put in another comment).
